src/main/java/no/nav/finnkandidatapi/permittert/PermittertArbeidssokerRepository.java

```diff
@@ -88,4 +91,8 @@ public Integer lagrePermittertArbeidssoker(PermittertArbeidssoker permittertArbe
     public void slettAllePermitterteArbeidssokere() {
         jdbcTemplate.execute("DELETE FROM permittert" );
     }
+
+    public List<PermittertArbeidssoker> hentAllePermitterteArbeidssokere() {
```

Comment:
Dette kan bli veldig mange - en liste med 400 tusen. Det er kanskje såpass lite data at det går greit likevel? Eller bør vi ha en form for paging her?

Reply:
Det vet jeg faktisk ikke. Tenker det burde gå fint...
Er det enkelt å legge til paginering?